lib/kbsemi.gi

```diff
@@ -438,6 +438,8 @@ end);
 GAPKB_REW.MakeKnuthBendixRewritingSystemConfluent :=
   GKB_MakeKnuthBendixRewritingSystemConfluent;
 
+MakeReadOnly( GAPKB_REW );
```

Comment:
I am slightly concerned about using `MakeReadOnly` because it is not clearly recgonizabl as HPC-GAP specific. So perhaps we should wrap this in `if IsBound(HPCGAP)` ?